myning/chapters/mine/__init__.py

```diff
@@ -54,8 +57,10 @@ def pick_time(mine: Mine):
         player.level * 4,
         player.level * 8,
     ]
-    options: list[Option] = [(f"{m} minutes", partial(start_mine, mine, m)) for m in minutes]
-    options.append(("Go Back", pick_mine))
+    options = [
+        Option(f"{m} minutes", partial(start_mine, mine, m), enable_hotkeys=True) for m in minutes
```

Comment:
for feature parity we have always been disabling hotkeys for minutes, but I'm open to discussion

Reply:
I don't have a strong opinion. I just saw it was a fixed list with somewhat consistent hotkeys, but I think feature parity is good.

Reply:
Yeah, I guess m i n u t can be hotkeys. I also don't have a strong opinion on this menu so we can do feature parity or the new default behavior. But we should remove the `minutes` condition from the `get_hotkey` function, this is a good improvement to avoid hardcoding that

Reply:
I disabled this.